doc/development/deprecations.rst

```diff
@@ -0,0 +1,83 @@
+.. _deprecations:
+
+Deprecations
+============
+
+Pending Deprecations
+--------------------
+
+* In-place inversion is now deprecated. This includes ``op.inv()`` and ``op.inverse=value``. Please
+  use ``qml.adjoint`` or ``qml.pow`` instead. 
+
+  - Still accessible in v0.27 and v0.28
+  - Removed in v0.29
```

Comment:
They are not yet removed.  They are just deprecated.  We will remove them for v0.29.

Reply:
😅 I feel like the bullet points read nicely as key results (or objectives), kind of like how we do OKRs. The exact timing of everything is addressed by the bullet points. Does that make sense? If this doesn't jive well with you, all good!

Reply:
Maybe I can replace "removed" everywhere with "deprecated". Is that better?

Reply:
```suggestion
* * In-place inversion — ``op.inv()`` and ``op.inverse=value`` — is deprecated. Please
use ``qml.adjoint`` or ``qml.pow`` instead. 

  - Still accessible in v0.27 and v0.28
  - Removed in v0.29
```

Reply:
Minor - saying 'Removed in v0.29' sounds odd to me, since this is past tense but it hasn't happened yet. Someone reading this could get confused, and think it has already been removed and we are at >v0.29.

I suggest using future tense here: 'To be removed in v0.29'.

Reply:
```suggestion
* In-place inversion — ``op.inv()`` and ``op.inverse=value`` — is deprecated. Please
use ``qml.adjoint`` or ``qml.pow`` instead. 

  - Still accessible in v0.27 and v0.28
  - Will be removed in v0.29
```